SKFlatMaker/script/varlist_Electron.txt

```diff
@@ -67,3 +67,7 @@ double	electron_puChMiniIso	1
 double	electron_trackIso	1
 double	electron_dr03EcalRecHitSumEt	1
 double	electron_dr03HcalDepth1TowerSumEt	1
+double  electron_dr03HcalTowerSumEt       1
+double  electron_dr03TkSumPt       1
+double  electron_ecalPFClusterIso       1
+double  electron_hcalPFClusterIso       1
```

Comment:
use 'tab' not spaces for these lines. I could not use space becuase of "unsigned int".

Reply:
This gives an error when running script/check_varlist.py